test-network-function/certification/suite.go

```diff
@@ -41,43 +41,50 @@ var certAPIClient api.CertAPIClient
 
 var _ = ginkgo.Describe(common.AffiliatedCertTestKey, func() {
 	if testcases.IsInFocus(ginkgoconfig.GinkgoConfig.FocusStrings, common.AffiliatedCertTestKey) {
+		return
+	}
+	testContainerCertificationStatus()
+	testOperatorCertificationStatus()
+})
 
-		// Query API for certification status of listed containers
-		ginkgo.When("getting certification status", func() {
+func testContainerCertificationStatus() {
+	// Query API for certification status of listed containers
+	ginkgo.When("getting certification status", func() {
+		ginkgo.It("get certification status", func() {
+			defer results.RecordResult(identifiers.TestContainerIsCertifiedIdentifier)
 			conf := configpkg.GetConfigInstance()
 			cnfsToQuery := conf.CertifiedContainerInfo
 			if len(cnfsToQuery) > 0 {
 				certAPIClient = api.NewHTTPClient()
-				for _, cnfRequestInfo := range cnfsToQuery {
-					cnf := cnfRequestInfo
+				for _, cnf := range cnfsToQuery {
+					cnf := cnf // pin
```

Comment:
Is this change intentional?

Reply:
Yes. Pinning objects  has something to do with keeping the object in fixed memory address.
My changes keeps the pinning. But , I'm not sure  if it's really necessary.